src/types.js

```diff
@@ -32,7 +32,7 @@ function toString(value) {
  * @return {boolean}
  */
 export function isArray(value) {
-  return toString(value) === '[object Array]';
+  return Array.isArray(value);
```

Comment:
looking at the history of the code, this was a refactor from xhr code which really didn't have a good reason to not use `Array.isArray` (especially if it is faster)